.github/workflows/develop.yml

```diff
@@ -11,10 +11,13 @@ jobs:
         steps:
             - name: Checkout code
               uses: actions/checkout@v2
-            - name: End-to-End tests
-              run: ./scripts/ci/end-to-end-tests.sh
```

Comment:
I realise you're trying to resolve the test failure that hit this PR, but it seems a bit strange to mix these workflow changes into this PR... Maybe let's separate it out...?

Reply:
Resolving, assuming these changes will be removed from here.